StRoot/StJetMaker/StJetMaker2012.cxx

```diff
@@ -20,12 +20,12 @@
 //#include "StjTPCRandomMuDst.h"
 #include "StjBEMCMuDst.h"
 #include "StjEEMCMuDst.h"
-#include "StjFMSMuDst.h"
+//#include "StjFMSMuDst.h"
```

Comment:
Instead of commenting out the lines just remove them. Hopefully, you have a good log message justifying the reason for that